content/academy/tutorials/dealing_with_dynamic_pages.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: Dealing with dynamic pages
 description: Learn about dynamic pages and dynamic content. How can we find out if a page is dynamic? How do we programmatically scrape dynamic content?
-menuWeight: 10
+menuWeight: 1
 paths:
-- web-scraping-for-beginners/crawling/dealing-with-dynamic-pages
+- tutorials/dealing-with-dynamic-pages
```

Comment:
Same with this. This is also integral part of scraping and not a standalone tutorial. It should be in some course.

That's why I said we should have the master sheet of content before we start moving things around...

Reply:
Same reason for moving this as above. This lesson is referenced throughout the academy, and I believe it is appropriate to put it into **Tutorials**. I only moved 2 lessons and migrated one into the Tutorials section so it wouldn't have just one lesson in there.